client/extensions/woocommerce/index.js

```diff
@@ -10,7 +10,7 @@ import config from 'config';
  */
 import { navigation, siteSelection } from 'my-sites/controller';
 import { renderWithReduxStore } from 'lib/react-helpers';
-import ProductForm from './app/products/product-form';
+import ProductCreate from './app/products/product-create';
```

Comment:
Given that each is its own route, it might make sense to have two separate container objects: one for create and one for update. This could be how we facilitate different text around it ( "Create a new product!" vs. "Updating 'Love Potion No. 9'")

It doesn't give me heartburn necessarily to have two components for this, as long as they're not repeating a bunch of code and are doing something worthwhile. That said, at this point, there's nothing in ProductCreate that couldn't apply to an Update as well, so I don't know that it really matters right now, even if it does later.